### DISCUSSION TITLE
GitHub student benefits verification

### DISCUSSION THREAD

**Discussion by snasif** [community/community#53838] [GitHub Education]
### Hi everyone,

I'm trying to verify my GitHub student benefits and keep getting rejected for the following reasons:

You are unlikely to be verified until you have completed your GitHub user profile with your full name as it appears in your academic affiliation document plus a short bio.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I'm an online student and I've updated my profile mulitple times. Not sure what else to do. Please someone help me. Thank you

**Comment by bbkx226** [comment]
Hi, from my experience, you can try out these three steps. First, double-check your profile information. Make sure your full name in your GitHub profile matches your name in your academic affiliation documentation exactly. Also, ensure that your bio includes some information about your academic pursuits and your school. Second, contact your school. Reach out to your school's registrar's office or student services department and request a letter or document stating that you are a distance learner. Lastly, if none of these works, contact GitHub support. If you have completed the above steps and are still having trouble verifying your eligibility, you can contact GitHub support directly. They may be able to provide additional guidance or assistance in verifying your status as a student.

**Reply by snasif** [reply]
Thank you for your help. I tried all of the above, I contacted support and opened a ticket and they kept closing them and telling me to refer to GitHub community discussions.
